modules/solid_properties/doc/content/bib/solid_properties.bib

```diff
@@ -0,0 +1,114 @@
+@Article{snead,
```

Comment:
Is the MooseDocs .bib file used for anything else in BISON (like in reports/manuals/etc.)? Or would it be possible to just delete the reference in BISON and have MooseDocs find the one in MOOSE?

Reply:
The way it is named I wouldn't know that the reference existed in MOOSE already because we use a different convention for naming the references (authorYearFirstPage#, which is the convention Elsevier uses if you download .bib citations).  I was just asking more in general if there is a convention we should all follow so that duplicates might be detected and they could be removed from the app. That being said, there is a flag in the documentation config file in the apps to simply report duplicate entries as a warning, which is probably fine.

Reply:
Circling back on this - @gambka are there any changes you would like incorporated regarding references for this PR?

Reply:
I thought about this a little more: I think we should just not worry about this situation. I don't think it's worth the effort of trying to enforce a bibtex key convention on all of MOOSE and its modules. Applications will sometimes use the same references and have a different key, and that's ok - no harm done.